shared/actions/chat2/index.tsx

```diff
@@ -2430,6 +2406,19 @@ const navigateToThread = (state: TypedState) => {
   return navigateToThreadRoute(state.chat2.selectedConversation)
 }
 
+const ensureSelectedMeta = (state: TypedState) => {
+  const meta = state.chat2.metaMap.get(state.chat2.selectedConversation)
+  if (!meta) {
```

Comment:
:sheep: in general i think `return false` is nicer looking than `return undefined`
you can also just do
`return meta && Chat2Gen.createMetaRequestTrusted...` since that'll be false or the action which types fine

Reply:
K there is a ton of `return undefined` in this file, so I thought that was the right way.